plugins/community/stephenh-ts-proto/v1.161.1/buf.plugin.yaml

```diff
@@ -0,0 +1,9 @@
+version: v1
+name: buf.build/community/stephenh-ts-proto
+plugin_version: v1.161.1
```

Comment:
This would be the 40th version in <1 yr. Should we consider skipping patches? @pkwarren